desktop/main.js

```diff
@@ -522,6 +537,8 @@ const mainWindow = () => {
 
                 ipcMain.on(ELECTRON_EVENTS.LOCALE_UPDATED, (event, updatedLocale) => {
                     Menu.setApplicationMenu(Menu.buildFromTemplate(localizeMenuItems(initialMenuTemplate, updatedLocale)));
+                    disposeContextMenu();
+                    disposeContextMenu = createContextMenu(updatedLocale);
```

Comment:
Why duplicated calls?

Reply:
First line will dispose the old context menu. Second line will reassign it. Is there any alternative solution you'd like to suggest?

Reply:
What's the root cause? Why not work on the first call?

Reply:
The first call will initialize the context menu. This second call will run whever the user changes the language

Reply:
ok makes sense